src/node_errors.cc

```diff
@@ -140,8 +140,43 @@ static std::string GetErrorSource(Isolate* isolate,
   return std::string(buf);
 }
 
+void PrintStackTrace(Isolate* isolate) {
+  Local<Context> context = isolate->GetCurrentContext();
+  Local<String> error_string =
+      String::NewFromOneByte(isolate,
+                             reinterpret_cast<const uint8_t*>("Error"),
+                             v8::NewStringType::kInternalized,
+                             5)
+          .ToLocalChecked();
+  Local<String> stack_trace_limit_string =
+      String::NewFromOneByte(
+          isolate,
+          reinterpret_cast<const uint8_t*>("stackTraceLimit"),
+          v8::NewStringType::kInternalized,
+          15)
+          .ToLocalChecked();
+  MaybeLocal<Value> stack_trace_limit_val =
+      context->Global()
+          ->Get(context, error_string)
+          .ToLocalChecked()
```

Comment:
This is not a valid `.ToLocalChecked()`, can you handle errors here (and for the `.ToChecked()` below, as @lundibundi pointed out)? Ideally, this block here would also be wrapped in a `v8::TryCatch` in order to swallow these errors from trying to get the stack trace